HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponentsController+Helpers.swift

```diff
@@ -0,0 +1,806 @@
+            }
+        }
+    }
+    
+    /**
+     Retrieves the default installed payment provider, if available.
+     - Returns: a Payment Provider object.
+     */
+    func defaultInstalledPaymentProvider() -> PaymentProvider? {
+        savedPaymentProvider()
+    }
+    
+    func storeDefaultPaymentProvider(paymentProvider: PaymentProvider) {
+        do {
+            let encoder = JSONEncoder()
+            let data = try encoder.encode(paymentProvider)
+            UserDefaults.standard.set(data, forKey: Constants.kDefaultPaymentProvider)
+        } catch {
+            GiniUtilites.Log("Unable to encode payment provider: (\(error))", event: .error)
+        }
+    }
+    
+    func savedPaymentProvider() -> PaymentProvider? {
+        if let data = UserDefaults.standard.data(forKey: Constants.kDefaultPaymentProvider) {
+            do {
+                let decoder = JSONDecoder()
+                let paymentProvider = try decoder.decode(PaymentProvider.self, from: data)
+                if self.paymentProviders.contains(where: { $0.id == paymentProvider.id }) {
+                    return paymentProvider
+                }
+            } catch {
+                GiniUtilites.Log("Unable to decode payment provider: (\(error))", event: .error)
+            }
+        }
+        return nil
+    }
+    
+    func sortPaymentProviders() {
+        guard !paymentProviders.isEmpty else { return }
+        self.paymentProviders = paymentProviders
+            .filter { $0.gpcSupportedPlatforms.contains(.ios) || $0.openWithSupportedPlatforms.contains(.ios) }
+            .sorted {
+                // First sort by whether the app scheme can be opened
+                if $0.appSchemeIOS.canOpenURLString() != $1.appSchemeIOS.canOpenURLString() {
+                    return $0.appSchemeIOS.canOpenURLString() && !$1.appSchemeIOS.canOpenURLString()
+                }
+                // Then sort by the index if the app scheme condition is the same
+                return ($0.index ?? 0) < ($1.index ?? 0)
+            }
+    }
+    
+    
+    // MARK: - Bottom sheets
+    
+    /**
+     Provides a custom Gini for the payment view that is going to be presented as a bottom sheet.
+
+     - Parameter documentId: An optional identifier for the document associated with the payment.
+     - Returns: A configured `UIViewController` for displaying the payment bottom view.
+     */
+    public func paymentViewBottomSheet(documentId: String?) -> UIViewController {
+        previousPresentedView = .paymentComponent
+        let paymentComponentBottomView = PaymentComponentBottomView(paymentView: paymentView(), bottomSheetConfiguration: configurationProvider.bottomSheetConfiguration)
+        return paymentComponentBottomView
+    }
+    
+    /**
+     Provides a custom Gini view that contains more information, bank selection if available and a tappable button to pay the document/invoice
+
+     - Parameters:
```

Comment:
[suggestion] I think we can remove parameters here since we don't have any. And the return  make can be: " - Returns: A `UIView` instance representing the payment component."